Userland/Libraries/LibWeb/HTML/Parser/HTMLParser.cpp

```diff
@@ -2976,8 +2999,25 @@ void HTMLParser::process_using_the_rules_for_foreign_content(HTMLToken& token)
 
     if (token.is_end_tag() && current_node().namespace_() == Namespace::SVG && current_node().tag_name() == SVG::TagNames::script) {
     ScriptEndTag:
+        // Pop the current node off the stack of open elements.
         (void)m_stack_of_open_elements.pop();
+        // Let the old insertion point have the same value as the current insertion point.
+        m_tokenizer.store_insertion_point();
+        // Let the insertion point be just before the next input character.
+        m_tokenizer.update_insertion_point();
+        // Increment the parser's script nesting level by one.
+        increment_script_nesting_level();
+        // Set the parser pause flag to true.
+        m_parser_pause_flag = true;
         TODO();
```

Comment:
Is this algorithm/step fully implemented? Leaving the TODO() here will crash the LibWeb user.

Reply:
No it is not. The TODO concerns scripts inside of SVG images.

I only changed this part as I implemented all points in the html spec where i could find a mention of the insertion point.

Reply:
Gotcha. Might be reasonable to annotate it with a FIXME comment to handle SVG scripts properly then.

Reply:
Done.